torch/csrc/api/include/torch/nn/modules/normalization.h

```diff
@@ -1,8 +1,8 @@
 #pragma once
 
-#include <torch/nn/cloneable.h>
 #include <torch/nn/cloneable.h>
 #include <torch/nn/functional/normalization.h>
+#include <torch/nn/functional/pooling.h>
```

Comment:
Is there a specific reason we need `#include <torch/nn/functional/pooling.h>` here?

Reply:
Yes, its note needed here. I shall remove this.